app/src/main/java/nerd/tuxmobil/fahrplan/congress/utils/FahrplanMisc.java

```diff
@@ -58,8 +59,8 @@ public static long getLectureStartTime(@NonNull Lecture lecture) {
         if (lecture.dateUTC > 0) {
             when = lecture.dateUTC;
         } else {
-            Time time = lecture.getTime();
-            when = time.normalize(true);
+            Clock time = lecture.getClock();
+            when = time.toMilliseconds();
```

Comment:
# Naming
Please find a self explaining name for the variable. Once `getClock()` is renamed as suggested in an earlier comment then it should be easy to replace `time` with a more descriptive variable name.